modules/apps/transfer/internal/packet_test.go

```diff
@@ -43,7 +41,7 @@ func TestUnmarshalPacketData(t *testing.T) {
 							Denom:  types.NewDenom("atom", types.NewHop("transfer", "channel-0")),
 							Amount: "1000",
 						},
-					}, sender, receiver, "", emptyForwardingPacketData)
+					}, sender, receiver, "", types.ForwardingPacketData{})
```

Comment:
I couldn't use the testing value because it was complaining about some import cycle...

Reply:
Yeah, I also was surprised, but decided to skip this rabbit hole...